styles.css

```diff
@@ -77,28 +96,404 @@ p {
   justify-content: flex-start;
 }
 
-.hero-text h1 {
+.about-text h1 {
   font-size: 40px;
+  margin-top: -10px;
+}
+
+.banner .banner-text h1 {
+  width: 95%;
   font-weight: 700;
+  font-size: 40px;
   line-height: 52px;
   color: #172b4d;
-  width: 95%;
 }
 
-.link-text {
-  font-weight: 600;
-  color: #6070ff;
-}
-
-.hero-text p {
+.banner .banner-text p {
   font-weight: 400;
   font-size: 16px;
   line-height: 24px;
   color: #344563;
+  margin-top: 10px;
+  margin-bottom: 12px;
+}
+
+/* Connect style */
+.link {
+  font-weight: 500;
+  font-size: 16px;
+  color: #7f8cff;
 }
 
+/* Social Media styles */
 .social {
   display: flex;
+  margin-top: 12px;
   gap: 20px;
+}
+
+.social-nl a {
+  margin-right: 10px;
+  color: #505f79;
+  font-size: 25px;
+}
+
+.social a:hover {
+  color: #505f79;
+  opacity: 0.5;
+}
+
+/* Work  */
+.work {
+  display: grid;
+  grid-template-columns: 1fr;
+  margin-top: 50px;
+}
+
+.social-nl a:hover {
+  color: #505f79;
+  opacity: 0.5;
+}
+
+.work .lang a {
+  padding: 5px 10px;
+  margin-right: 10px;
+  background: #ebebff;
+  font-weight: 500;
+  font-size: 12px;
+  color: #6070ff;
+  border-radius: 8px;
+}
+
+.work .card {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  border-radius: 16px;
+  padding: 20px;
+  margin-top: 104px;
+  border: 1px solid #e6e8eb;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.1);
+}
+
+.lan {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 0;
+}
```

Comment:
- [ ] Good job so far 👏 However, as suggested, please make sure to adjust your styling including margin and element alignment for the about section. Kindly add a bit more space around the list items. Also, the arrows are slightly inwards according to the Figma template. And align `javascript`, `html`, `css`  to the center according to the previous reviewer's suggestion


Your design | Given design
---| ---|
![image](https://github.com/Srilearns/Portfolio-Website-Mobile-First/assets/106119105/73e3cfbd-ff4c-44fb-afc7-45ad2f492d17) | ![image](https://github.com/Srilearns/Portfolio-Website-Mobile-First/assets/106119105/d20becd8-69bf-459a-999a-61aab6e978da)

Reply:
Fixed.